Print_A_Bot/djwifi/templates/djwifi/includes/handlebars_scripts.html

```diff
@@ -0,0 +1,47 @@
+{% load i18n %}
```

Comment:
collection of "handlebars templates" and functions that the djwifi page requires to function